src/Pie.js

```diff
@@ -41,10 +41,9 @@ const RoundDividers = ({ paintedSections, dividerSize, width, radius, background
   let dividerColorOverlayArray = [];
   let dividerArray = [];
   paintedSections.forEach((section, index) => {
-    const { percentage, color, startAngle } = section;
-    const shouldShowDividerForSection = shouldShowDivider(percentage, dividerSize);
-
-    if (shouldShowDividerForSection) {
+    const { color, startAngle } = section;
+    
+    if (paintedSections.length > 1) {
```

Comment:
We should still have the dividerSize check here otherwise it will still do this when `dividerSize` is 0 (no dividers). For simplicity see my below comment

Reply:
This condition should be checked before looping through `paintedSections`.